x-pack/plugins/serverless_search/public/application/components/overview_panels/select_client.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiCallOut,
+  EuiFlexGrid,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+import { PLUGIN_ID } from '../../../../common';
+
+import { useKibanaServices } from '../../hooks/use_kibana';
+import { LanguageDefinition, Languages } from '../languages/types';
+import { OverviewPanel } from './overview_panel';
+import { docLinks } from '../../../../common/doc_links';
+interface SelectClientProps {
+  setSelectedLanguage: (language: LanguageDefinition) => void;
+  languages: LanguageDefinition[];
+  selectedLanguage: LanguageDefinition;
+}
+export const SelectClientPanel: React.FC<SelectClientProps> = ({
+  setSelectedLanguage,
+  languages,
+  selectedLanguage,
+}) => {
+  const { http } = useKibanaServices();
+  const { euiTheme } = useEuiTheme();
+  const panelItems = languages.map((language) => (
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem grow={false}>
+        <EuiPanel
+          hasBorder
+          borderRadius="m"
+          onClick={() => setSelectedLanguage(language)}
+          grow={false}
+          paddingSize="m"
+          css={
+            selectedLanguage === language
+              ? css`
+                  border: 1px solid ${euiTheme.colors.primary};
+                `
+              : css`
+                  border: 1px solid ${euiTheme.colors.lightShade};
+                `
+          }
+          color={selectedLanguage === language ? 'primary' : 'plain'}
+        >
+          <EuiFlexGroup direction="column" gutterSize="l" justifyContent="center">
+            <EuiFlexItem>
+              <EuiImage
+                alt=""
+                src={http.basePath.prepend(`/plugins/${PLUGIN_ID}/assets/${language.iconType}`)}
+                height="100px"
+                width="128px"
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText
+                textAlign="center"
+                color={selectedLanguage === language ? 'default' : 'subdued'}
+              >
+                <h5>
+                  <FormattedMessage
```

Comment:
This shouldn't be need with the name property already translated. Also the `defaultMessage` is just a variable so there is no need for a translation.

Reply:
Replaced with `i18n` translation.

Reply:
I see now, since name is already [translated](https://github.com/elastic/kibana/blob/514cc3f261e9830f0440aadc66f9003f6c87e693/x-pack/plugins/serverless_search/public/application/components/languages/curl.ts#L20). Just adding {language.name} should be good. Will revert my changes.